docs/make.jl

```diff
@@ -22,6 +22,8 @@ makedocs(
         "Introduction" => "index.md",
         "User Guide" => Any[
             "Getting Started" => "man/getting_started.md",
+            "Working with DataFrames" => "man/Working_with_DataFrames.md",
```

Comment:
before merging please do not use capital letters in file names for consistency.

Reply:
Resolved.